vnext/ReactUWP/Utils/PropertyUtils.h

```diff
@@ -174,26 +174,47 @@ bool TryUpdateBorderProperties(
       UpdateControlBorderResourceBrushes(element, nullptr);
     }
   } else if (propertyName == "borderLeftWidth") {
-    if (propertyValue.isNumber())
+    if (propertyValue.isNumber()) {
       SetBorderThickness(node, element, ShadowEdges::Left, propertyValue.asDouble());
+    } else if (propertyValue.isNull()) {
+      SetBorderThickness(node, element, ShadowEdges::Left, 0);
+    }
   } else if (propertyName == "borderTopWidth") {
-    if (propertyValue.isNumber())
+    if (propertyValue.isNumber()) {
       SetBorderThickness(node, element, ShadowEdges::Top, propertyValue.asDouble());
+    } else if (propertyValue.isNull()) {
+      SetBorderThickness(node, element, ShadowEdges::Top, 0);
+    }
```

Comment:
Could replace all of this with something like this:
```
std::optional<ShadowEdges> edge;
if (propertyName == "borderTopWidth") {
  edge = ShadowEdges::Top
} else if (propertyName == "borderRightWidth") {
```
and at the end
```
if (edge.has_value()) {
  if (propertyValue.isNumber()) {
    SetBorderThickness(node, element, edge.value(), propertyValue.asDouble());
  } else if (propertyValue.isNull()) {
    SetBorderThickness(node, element, edge.value(), 0);
  }
}
```
And/or put the mapping of string -> edge into a map.

Reply:
Updated with map, thanks for the tips on std::optional..

---
In reply to: [357864798](https://github.com/microsoft/react-native-windows/pull/3754#discussion_r357864798) [](ancestors = 357864798)